eventum/authorize.py

```diff
@@ -0,0 +1,30 @@
+import eventum
+import argparse
+from sys import argv
+
+from oauth2client.file import Storage
+from oauth2client.client import flow_from_clientsecrets
+from oauth2client.tools import run_flow
+from oauth2client import tools
+
+# from config import flask_config
+if(len(argv)) != 3:
```

Comment:
unneeded parens.  Also, we should only check if the `len(argv) < 3`, in case they pass extra arguments for the Google Argparse stuff.